blockstore/splitstore/README.md

```diff
@@ -0,0 +1,66 @@
+# SplitStore: An actively scalable blockstore for the Filecoin chain
+
+The SplitStore was first introduced in lotus v1.5.1, as an experiment
+in reducing the performance impact of large blockstores.
+
+With lotus v1.11.1, we introduce the next iteration in design and
+implementation, which we call SplitStore v1.
+
+The new design (see [#6474](https://github.com/filecoin-project/lotus/pull/6474)
+evolves the splitstore to be a freestanding compacting blockstore that
+allows us to keep a small (60-100GB) working set in a hot blockstore
+and reliably archive out of scope objects in a coldstore.  The coldstore
+can be a noop store, whereby out of scope objects are discarded or a
+regular badger blockstore (the default), which can be periodically
+garbage collected according to configurable user retention policies.
+
+To enable the splitstore, edit `.lotus/config.toml` and add the following:
+```
+[Chainstore]
+  EnableSplitstore = true
+```
+
+If you want to use the noop coldstore, also add the following:
```

Comment:
Can we change this to reflect intent?

1. The user doesn't care about what happens under the covers.
2. If the option specifies _intent_, we can _change_ what happens under the covers without messing with the options.

Reply:
Ok. Any suggestions for wording?

Reply:
I'm not entirely sure why a user might want to do this, but I _assume_ it's because they're going to put a snapshot there but don't want to keep old state otherwise? Unclear?

Reply:
If they want to run with not much of a disk, it is very handy; it is also critical for the boosters.

Reply:
Updated the wording in https://github.com/filecoin-project/lotus/pull/6474/commits/b9a5ea8f7bed48942e85dbf0fae0976402bb4f6b

Does it look better?